### ISSUE TITLE
Add locked amount to the balance on the dashboard page

### ISSUE THREAD

**Issue by punkrock** [LiskHQ/lisk-desktop#3671]
The dashboard is the first thing a user sees if it logs in. It would be good, to prevent heart attacks or something similar, that the locked amount that was used for voting is also shown, like it is already on the "wallet" page.

![grafik](https://user-images.githubusercontent.com/1260700/124288647-b9255e80-db51-11eb-9638-0f8f141f6998.png)

**Comment by reyraa** [comment]
Good point @punkrock . I think we can use the opportunity to re-imagine this component to provide more useful information to the user.
Back then the summary of an account was limited to the balance. Now we have

- available and locked balance,
- delegacy info,
- Multisignature / regular accounts,
- and soon, we'll have pending multisignature transactions which require user's action

and much more.
